app/services/facet.service.js

```diff
@@ -129,22 +132,22 @@
       for (var i in this.facets[key]) {
         var result;
         var filter = this.facets[key][i].value;
-        // filter is a function
         if (filter.call) {
+          // filter is a function
```

Comment:
I find putting this above `if (filter.call)` is clearer, since it makes it clear why that conditional's being used.